### PR TITLE
Updates captainplanet test scripts to use /tmp/capt for test files

### PR DESCRIPTION
**Pull request by dylanlott**

### CODE REVIEW

File: scripts/test-captplanet.sh [hunk 1]
```diff
@@ -13,30 +13,36 @@ kill -9 $CAPT_PID
 captplanet run &
 CAPT_PID=$!
 
+#setup tmpdir for testfiles and cleanup
+TMP_DIR=/tmp/capt
```
Comment (egonelbre) [inline scripts/test-captplanet.sh]: Depending on the system the temp directory might be somewhere else. Although probably won’t cause too much problems.

**Comment by stefanbenten** [comment]
@dylanlott This script wont run on Windows with that path.

File: scripts/test-captplanet.sh (hunk 1 above)
Comment (stefanbenten) [reply scripts/test-captplanet.sh]: We should switch depending on the OS to another path

Comment (dylanlott) [reply scripts/test-captplanet.sh]: I thought about this, but I didn't know if we wanted to try and support windows. It's pretty easy to change this to be just the current directory.

Comment (jtolds) [reply scripts/test-captplanet.sh]: if you use `$(mktemp -d)` you should get a cross-platform tmp directory. you can also set an exit trap to clean up the directory after the test ends http://redsymbol.net/articles/bash-exit-traps/

Comment (dylanlott) [reply scripts/test-captplanet.sh]: so it would be `TMP_DIR=$(mktemp -d) like that?

**Comment by dylanlott** [comment]
@stefanbenten updated to use `mktemp`

File: scripts/test-captplanet.sh (hunk 1 above)
Comment (dylanlott) [reply scripts/test-captplanet.sh]: Updated the script to use `mktemp` for both the tmp and cmp directories

**Pull request merged**